app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<main>
+  <section>
+    <ul>
+      <% @users.each do |user|%>
+        <%= link_to user do %>
+          <li class="card hover">
+            <div class="user-image border"></div>
```

Comment:
- [Optional] I think user image was supposed to be retrieved from the db and display here instead of styling this div :+1: